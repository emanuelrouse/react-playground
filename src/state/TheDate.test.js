import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import TheDate from './TheDate';

describe('TheDate component', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<TheDate />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it.skip('Renders UI as expected', () => {
    const tree = renderer
        .create(<TheDate />)
        .toJSON();
    expect(tree).toMatchSnapshot();
  })
})