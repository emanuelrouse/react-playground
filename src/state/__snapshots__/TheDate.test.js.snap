// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheDate component Renders UI as expected 1`] = `
<div>
  10/12/2019, 3:29:49 AM
</div>
`;
